documentscanner/src/main/res/values/strings.xml

```diff
@@ -1,24 +1,26 @@
-    <string name="retake_action">Retake</string>
     <string name="general_add">Add</string>
     <string name="save_action">Save</string>
-    <string name="file_type_pdf">PDF</string>
-    <string name="file_type_jpg">JPG</string>
-    <string name="quality_high">High</string>
-    <string name="quality_medium">Medium</string>
-    <string name="quality_low">Low</string>
-    <string name="invalid_input">Invalid input</string>
-    <string name="dialog_discard_scan_title">Discard scan?</string>
-    <string name="dialog_discard_scan_body">The current scan will be lost</string>
-    <string name="dialog_discard_all_scans_title">Discard all scans?</string>
-    <string name="dialog_discard_all_scans_body">The current document will be lost</string>
-    <string name="dialog_delete_scan_body">Delete current scan?</string>
+    <string name="format_page_count">%1$d / %2$d</string>
+    <string name="action_retake" context="Action text to retake current scan picture">Retake</string>
+    <string name="title_crop" context="Screen title for cropping current scan">Crop</string>
+    <string name="title_save_scan" context="Screen title for saving current scan">Save scan</string>
+    <string name="scan_file_type" context="Input field description in the document save screen">File type</string>
+    <string name="scan_quality" context="Input field description in the document save screen">Quality</string>
+    <string name="scan_destination" context="Input field description in the document save screen">Destination</string>
+    <string name="scan_file_type_pdf" context="Document scanner PDF file type option">PDF</string>
+    <string name="scan_file_type_jpg" context="Document scanner JPG file type option">JPG</string>
+    <string name="scan_quality_high" context="Document scanner high quality option">High</string>
+    <string name="scan_quality_medium" context="Document scanner medium quality option">Medium</string>
+    <string name="scan_quality_low" context="Document scanner low quality option">Low</string>
+    <string name="scan_dialog_discard_title" context="Title text shown on document scanner dialog to discard scan">Discard scan?</string>
+    <string name="scan_dialog_discard_body" context="Body text shown on document scanner dialog to discard scan">The current scan will be lost</string>
+    <string name="scan_dialog_discard_all_title" context="Title text shown on document scanner dialog to discard scan all scans">Discard all scans?</string>
+    <string name="scan_dialog_discard_all_body" context="Body text shown on document scanner dialog to discard all scans">The current document will be lost</string>
+    <string name="scan_dialog_delete_body" context="Body text shown on document scanner dialog to delete current scan">Delete current scan?</string>
+    <string name="scan_requires_permission" context="Message shown when users deny camera permissions to scan documents">Camera permissions are needed to scan documents</string>
 </resources>
```

Comment:
Rename this file by `strings_document_scanner.xml` in order to upload it to Babel. The filename `strings.xml` is already in use in Babel for the strings of the Android app.